configs/mender_convert_config

```diff
@@ -78,6 +78,11 @@ MENDER_DATA_PART_GROWFS=y
 # input disk image
 MENDER_DEVICE_TYPE=""
 
+# A string that defines the filenames of the deployed files.
+# This is optional. If no FILE_NAME is defined the filenames will be a combination off
+# the date and the names of all DEVICE_TYPES. These will be seperated with a +.
+FILE_NAME=""
```

Comment:
Technically, it is not used a a "file" name but as an "image" name, as mender-convert will be appending a `.img` and creating other related files like the Artifact or the tests configurations using this "image" name as a base name.

I suggest `IMAGE_BASE_NAME` or even better `DEPLOY_IMAGE_NAME` or `OUTPUT_IMAGE_NAME`.

Reply:
Thanks for good feed back Lluís, I appreciate it!